README.md

```diff
@@ -1,3 +1,6 @@
+
+[![Build Status](https://travis-ci.org/scalaj/scalaj-http.png)](https://travis-ci.org/ASIDataScience/scala-plotly-client)
```

Comment:
The Build Status link should be updated to point to https://travis-ci.org/ASIDataScience/scala-plotly-client.png.